rest/indextest/index_init_api_test.go

```diff
@@ -227,22 +227,6 @@ func TestChangeIndexPartitionsErrors(t *testing.T) {
 	}
 }
 
-func TestChangeIndexPartitionsSameNumber(t *testing.T) {
-	if base.UnitTestUrlIsWalrus() || base.TestsDisableGSI() {
-		t.Skip("This test only works against Couchbase Server with GSI enabled")
-	}
-
-	// requires index init
-	base.LongRunningTest(t)
-
-	rt := rest.NewRestTester(t, &rest.RestTesterConfig{DatabaseConfig: &rest.DatabaseConfig{DbConfig: rest.DbConfig{Index: &rest.IndexConfig{NumPartitions: base.Ptr(uint32(2))}}}})
-	defer rt.Close()
-
-	resp := rt.SendAdminRequest(http.MethodPost, "/{{.db}}/_index_init", `{"num_partitions":2}`)
```

Comment:
Should be able to move this `2 -> 2` use-case into a `TestChangeIndexPartitions` subtest now just for coverage.